.idea/misc.xml

```diff
@@ -1,29 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="NullableNotNullManager">
```

Comment:
let's keep all this project config :)

Reply:
I think these got messed up during my re-download of AS 🤔 removing them all